### ISSUE TITLE
Error: Could not locate the bindings file

### ISSUE THREAD

**Issue by agustin-eticco** [odoo-ide/vscode-odoo#11]
First of all thanks for the work you are doing for the extension, it will be really useful for Odoo development.

After installing it in a WSL environment it gives me the following errors, I was looking at the paths and the only one that matches is /lib/ but I don't have node-v93, I have node-v106 and node-v110.

```
Error: Could not locate the bindings file. Tried:
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/build/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/build/Debug/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/build/Release/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/out/Debug/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/Debug/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/out/Release/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/Release/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/build/default/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/compiled/16.14.2/linux/x64/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/addon-build/release/install-root/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/addon-build/debug/install-root/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/addon-build/default/install-root/better_sqlite3.node
 → /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/lib/binding/node-v93-linux-x64/better_sqlite3.node
    at t (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1509396)
    at new c (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1495227)
    at new t.OdooIndexDbSqlite (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1448266)
    at Function.getInstance (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1457051)
    at new _ (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1487274)
    at e (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1470126)
    at t.run (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1276914)
    at Object.11028 (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:1:1470121)
    at __webpack_require__ (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:24:245761)
    at /home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:24:246336
    at Object.<anonymous> (/home/agustin/.vscode-server/extensions/trinhanhngoc.vscode-odoo-0.2.0-linux-x64/dist/server.js:24:246399)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
[Error - 7:35:31 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 7:35:31 AM] The Odoo server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
[Error - 7:35:31 AM] Odoo client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 7:35:31 AM] Restarting server failed
  Message: Pending response rejected since connection got disposed
  Code: -32097 
```

I also tried it on a Windows project and I get the following error:
```
[Info  - 7:54:00 AM] Pyright language server 1.1.304 starting
[Info  - 7:54:00 AM] Server root directory: c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist
[Info  - 7:54:00 AM] Data storage path: /C:/Users/agust/AppData/Roaming/Code/User/workspaceStorage/d71154a8632662ecf732042fef1a142e/trinhanhngoc.vscode-odoo
Error: ENOENT: no such file or directory, mkdir '/C:/Users/agust/AppData/Roaming/Code/User/workspaceStorage/d71154a8632662ecf732042fef1a142e/trinhanhngoc.vscode-odoo'
    at mkdirSync (node:fs:1336:3)
    at Object.e.mkdirSync (node:electron/js2c/asar_bundle:5:12607)
    at _prepareDataStorage (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:1:1487566)
    at new _ (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:1:1487233)
    at e (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:1:1470126)
    at t.run (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:1:1276914)
    at Object.11028 (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:1:1470121)
    at __webpack_require__ (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:24:245761)
    at c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:24:246336
    at Object.<anonymous> (c:\Users\agust\.vscode\extensions\trinhanhngoc.vscode-odoo-0.2.0-win32-x64\dist\server.js:24:246399)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
[Error - 7:54:00 AM] Server initialization failed.
```

Is there anything I need to install or configure? I have already changed the Python Language Server to None.

Thanks in advance!

**Comment by trinhanhngoc** [comment]
Hi @agustin-eticco , I need you to provide more information:

1. Your VSCode version. You can open the VSCode's About dialog to copy detailed information like:
```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:41:48.638Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: Yes
```

2. Your WSL environment information: OS, version,...

**Comment by agustin-eticco** [comment]
@trinhanhngoc 

```
Version: 1.77.3 (system setup)
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:02.548Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
```

Im running WSL v2:
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

**Comment by trinhanhngoc** [comment]
@agustin-eticco 

There are actually two problems here:

- The Odoo extension does not support running in remote environment yet.
- Error when running on Windows. I just released a new version to fix this error. Please update it and check if it works for you.

**Comment by trinhanhngoc** [comment]
Hi @agustin-eticco I just published a new version (0.2.2) of the Odoo extension with support for remote development environments (WSL, Docker, ...). You can update it now.

**Comment by agustin-eticco** [comment]
@trinhanhngoc i was testing on WSL and all available feature are working, thanks!
